java/src/test/resources/testdata

```diff
@@ -0,0 +1 @@
+../../../../test/testdata/
```

Comment:
I agree on this not making me happy, none of the three different solutions are very clean.

I went for adding an antrun execution, as the project already used that for protobuf.